src/CyPhyMasterInterpreter/Rules/StructuralFEATestBenchChecker.cs

```diff
@@ -56,7 +56,7 @@ protected IEnumerable<ContextCheckerResult> FaceConstructMustBeAbaqusModelBased(
                         {
                             var feedback = new ContextCheckerResult()
                             {
-                                Success = false,
+                                Success = true,

```

Comment:
The goal was to allow Face for all cases.  Before it was just model-based FEA.  

 

From: Kevin Smyth [mailto:notifications@github.com] 
Sent: Wednesday, December 02, 2015 3:34 PM
To: metamorph-inc/meta-core
Cc: Robert Owens
Subject: Re: [meta-core] CAD_002 Added FEA support for pressure loading and Face construct. M… (#5)

 

In src/CyPhyMasterInterpreter/Rules/StructuralFEATestBenchChecker.cs <https://github.com/metamorph-inc/meta-core/pull/5#discussion_r46478653> :

> @@ -56,7 +56,7 @@ protected IEnumerable<ContextCheckerResult> FaceConstructMustBeAbaqusModelBased(
>                          {
>                              var feedback = new ContextCheckerResult()
>                              {
> -                                Success = false,
> +                                Success = true,

Changing this to "Success = true" doesn't line up with the Message. Should this ContextCheckerResult be removed? i.e. is the Face construct allowed for ABAQUS deck based now?

—
Reply to this email directly or view it on GitHub <https://github.com/metamorph-inc/meta-core/pull/5/files#r46478653> . <https://github.com/notifications/beacon/ANMj_mpa3TP3a7kE2gy6RhDIy59gqG_qks5pL1tLgaJpZM4GtfK6.gif>